charts/emissary-ingress/templates/rbac.yaml

```diff
@@ -78,10 +78,15 @@ rules:
     resources:
     - namespaces
     - services
-    - secrets
     - endpoints
     verbs: ["get", "list", "watch"]
 
+  - apiGroups: [""]
+    resources:
+    - secrets
+    - customresourcedefinitions
+    verbs: ["get", "list", "watch", "update", "patch", "create", "delete" ]
```

Comment:
Can't we just give it permission to get/create/update the _one_ secret, instead of giving it everything for all of them?